docs/evm/cadence/direct-calls.md

```diff
@@ -25,16 +25,17 @@ let addr = EVM.EVMAddress(bytes: bytes)
 Once you have access to an `EVMAddress`, you can query various pieces of state information such as:
 
 - `balance() EVM.Balance` provides the balance of the address. It returns a balance object rather than a basic type to avoid errors when converting from flow to atto-flow.
-- `nonce() UInt64`  retrieves the nonce associated with the address.
+- `nonce() UInt64` retrieves the nonce associated with the address.
 - `code(): [UInt8]` fetches the code at the address; it returns the smart contract code if applicable, and is empty otherwise.
 
 ```
 import EVM from <ServiceAddress>
 
 access(all)
-fun main(bytes: [UInt8; 20]) {
+fun main(bytes: [UInt8; 20]): EVM.Balance {
     let addr = EVM.EVMAddress(bytes: bytes)
     let bal = addr.balance()
+    return bal
 }
```

Comment:
This change assumes that we want the script to return a usable value (which seems better to me?)